### PR TITLE
Whitelisting

### PR DESCRIPTION
**Pull request by shubhamgupta14cse**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,5 +3,4 @@ source 'http://rubygems.org'
 gem 'sinatra', '~>2.0.7'
 gem 'haml', '~>5.1.2'
 # Local
-# gem 'ruby-sdk', path: '/path/to/ruby-sdk/'
-gem 'ruby-sdk', '~1.3.0'
+gem 'vwo-sdk', path: '/Users/sahil/projects/wingify/vwo-ruby-sdk/'
```
Comment (softvar) [inline Gemfile:6]: this is wrong

File: Gemfile.lock [hunk 2]
```diff
@@ -1,3 +1,10 @@
+PATH
```
Comment (softvar) [inline Gemfile.lock:1]: why are we changing this file for?

File: config/vwo.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "account_id": "",
-  "sdk_key": "",
-  "ab_test_campaign_key": "",
+  "account_id": "89499",
```
Comment (softvar) [inline config/vwo.json:2]: this need not be committed ever

File: Gemfile.lock (hunk 2 above)
Comment (sahilbathla) [reply Gemfile.lock:1]: auto generated bro

File: config/vwo.json (hunk 3 above)
Comment (sahilbathla) [reply config/vwo.json:2]: yes

**Pull request closed without merging**